src/index.html

```diff
@@ -12,7 +12,138 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <div class="stars stars--0">
+      <a
+        href="#"
+        class="stars__star"
+      ></a>
```

Comment:
Consider using `div` for the star element or `ul` and `li` combination

Reply:
I thought of using other elements, but a tag made sense to me because it could potentially be used to navigate straight to reviews